blazor/common/accessibility.md

```diff
@@ -143,24 +143,47 @@ padding-top: 20px;
         <div class="controlanchorlink"><a target="_self" href="https://blazor.syncfusion.com/documentation/rich-text-editor/accessibility">RichTextEditor</a></div>
         <div class="controlanchorlink"><a target="_self" href="https://blazor.syncfusion.com/documentation/pdfviewer-2/keyboard-accessibility">PDF Viewer</a></div>
         <div class="controlanchorlink"><a target="_self" href="https://blazor.syncfusion.com/documentation/document-editor/accessibility">Word Processor</a></div>
+        <div class="controlanchorlink">Image Editor</a></div>
+        <div><p class="controlcategory">FILE FORMAT FRAMEWORKS</p></div>
```

Comment:
Can exclude this File Format frameworks?

Reply:
Hi @rajendranr-5483 ,

I removed File Format Framework section and committed.

Regards,
@Backiaraj